### PR TITLE
Adding demo section with Code block

### PR DESCRIPTION
**Pull request by MarcoABCorrea**

### CODE REVIEW

File: styles/components/_section.scss [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
     @include media('medium') {
       width: auto;
-      margin-right: 60px;
+      margin-right: var(--spacing-60);
```
Comment (sebastianbenz) [inline styles/components/_section.scss:40]: nit: encoding the value into the variable name kind of beats the purpose of using a variable. Would it make sense to use an abstraction here?

File: site/_includes/partials/demo.njk [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  <button class="bd-button--light --inverted --uppercase">animation</button>
+  <button class="bd-button--light --inverted --uppercase">accordion</button>
+  <button class="bd-button--light --inverted --uppercase">autocomplete</button>
+</div>
+
+<div class="bd-row__column --grow">
+  <div class="bd-demo-container">
+    <div class="bd-demo__code">
+{% highlight 'html' %}
+<head>
```
Comment (sebastianbenz) [inline site/_includes/partials/demo.njk:12]: inline `head` tag? Should this be an iframe instead?

**Review by sebastianbenz — COMMENTED** [review]
Can you please not commit .DS_Store files? Ideally, we should add these to the gitignore file (and remove the ones already in the repo).

File: site/_includes/partials/demo.njk (hunk 2 above)
Comment (MarcoABCorrea) [reply site/_includes/partials/demo.njk:12]: The head is part of a code block which is not rendered. This is part of a code snippet as you can see here:

image

**Comment by MarcoABCorrea** [comment]
> Can you please not commit .DS_Store files? Ideally, we should add these to the gitignore file (and remove the ones already in the repo).

DS_Store files removed

File: site/_includes/partials/demo.njk (hunk 2 above)
Comment (sebastianbenz) [reply site/_includes/partials/demo.njk:12]: 🤣 sorry, missed the `{% highlight 'html' %}`

File: styles/components/_section.scss (hunk 1 above)
Comment (MarcoABCorrea) [reply styles/components/_section.scss]: Added a fix for spacings naming

**Review by sebastianbenz — APPROVED** [review]
Thanks!

**Pull request merged**